pages/components/Toast/toast.css

```diff
@@ -0,0 +1,47 @@
+@import url(/css/main.css);
+
+.toast-preview-container {
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    border: 1px solid var(--secondary-faded-color);
+    border-radius: 8px;
+    padding: 1rem;
+    gap: 1rem;
+}
+
+.toast{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
You could use flex as helper class and it will avoid repeating code

Reply:
Thanks for the suggestion, that would add one more class to the element and increase specificity so keeping this way.